src/components/work/details/Details.js

```diff
@@ -165,7 +202,25 @@ export default function Wrap(props) {
     data.work.materialTypes.find((element) => element.materialType === type) ||
     data.work.materialTypes[0];
 
-  return <Details {...props} data={materialType?.manifestations?.[0]} />;
+  // pass subjects also - to show genre and form
+  const allsubjects = data?.work?.subjects;
+  // Include wanted subject types
+  const include = ["DBCO", "genre", null];
+  // filter out other subjects
+  const subjects = allsubjects.filter((s) => include.includes(s.type));
+  // filter out duplicates
+  const uniquesubjects = subjects.filter(
```

Comment:
vi har tidligere undladt at fjerne dubletter for at synliggøre at det skal håndteres længere nede i stakken